packages/synapse-react-client/src/lib/containers/UserCard.tsx

```diff
@@ -54,6 +54,7 @@ export type UserCardProps = {
   /** Whether to show the full name in the small user card */
   showFullName?: boolean
   className?: string
+  showAccountLevelIcon?: boolean
```

Comment:
UserCardSmall prop that was not exposed in `UserCard`.

Exposed here so we can use `UserCard` instead of `UserCardSmall` because `UserCard` will handle data fetching for us.